packages/loot-core/src/server/accounts/sync.ts

```
@@ -322,57 +326,83 @@
       trans.imported_payee = trans.imported_payee.trim();
     }
-
-    // It's important to resolve both the account and payee early so
-    // when rules are run, they have the right data. Resolving payees
-    // also simplifies the payee creation process
-    trans.account = acctId;
-    trans.payee = await resolvePayee(trans, trans.payeeName, payeesToCreate);
-
-    trans.cleared = Boolean(trans.booked);
-
+    
```

ESLint

[error] Delete `····`

(prettier/prettier)